raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java

```diff
@@ -414,7 +413,7 @@ private void insertRaster(String fileName, String absPath) throws SQLException,I
 		conn.commit();
 		createSchema.close();
 		ProcessBuilder pb =
-				new ProcessBuilder("/bin/sh", "-c", "raster2pgsql -R -I -C -M -t auto "+ absPath +" "+  targetTable + " | PGPASSWORD="+ dbUserPw +" psql -h db -U "+ dbUser +" -d sauber_data -v ON_ERROR_STOP=ON");
+				new ProcessBuilder("/bin/sh", "-c", "raster2pgsql -d -R -I -C -M -t auto "+ absPath +" "+  targetTable + " | PGPASSWORD="+ dbUserPw +" psql -h db -U "+ dbUser +" -d sauber_data -v ON_ERROR_STOP=ON");

```

Comment:
Just for clarification: what does `-d`  for  `raster2pgsql` do?

Reply:
So far we still import the raster both into the file system and database, the latter one being optional at this point. 
raster2pgsql creates a db table for each raster. To update outdated raster, `-d` deletes old table if exists.